projects/uswds-components/src/lib/form-field/form-field.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, Input, ChangeDetectorRef } from '@angular/core';
+import { NgControl } from '@angular/forms';
+
+@Component({
+  selector: 'uswds-form-field',
+  templateUrl: 'form-field.component.html',
+})
+export class FormFieldComponent {
+  /**
+   * sets the label text
+   */
+  @Input() public label: string;
+  /**
+   * sets the name attribute value
+   */
+  @Input() public name: string;
+  /**
+   * sets the description text
+   */
+  @Input() public description: string;
+  /**
+   * deprecated, toggles the required text
+   */
+  @Input() public required: boolean = false;
+  /**
+   * toggles the required text
+   */
+  @Input() public requiredFlag: boolean = false;
+
+  /**
+   * set the error message
+   */
+  @Input() public set errorMessage(message: string) {
+    this._errorMessage = message;
+  }
+
+  public get errorMessage(): string {
+    return this._errorMessage;
+  }
+
+  public input: HTMLElement;
+
+  public errorElId: string;
+  public descriptionElId: string;
+
+  private _errorMessage = '';
+
+  constructor(private cdr: ChangeDetectorRef) {}
+
+  public formatErrors(control: NgControl) {
+    if (!control) {
+      return;
+    }
+
+    if (control.pristine) {

```

Comment:
If we add an additional check here - `if (control.pristine || !control.errors)` - can we remove the else if down below on line 79?

Reply:
Removed